fcli-core/fcli-common/src/main/java/com/fortify/cli/common/cli/cmd/AbstractRunnableCommand.java

```diff
@@ -126,16 +126,16 @@ public static final class GenericOptionsArgGroup {
         @Option(names = {"-h", "--help"}, usageHelp = true)
         private boolean usageHelpRequested;
         
-        @Option(names = "--env-prefix", defaultValue = "FCLI_DEFAULT")
+        @Option(names = "--env-prefix", defaultValue = "FCLI_DEFAULT", paramLabel = "<prefix>")
         @Getter private String envPrefix;
         
-        @Option(names = "--log-file")
+        @Option(names = "--log-file", defaultValue = "./fcli.log")
```

Comment:
Adding a default value here is wrong, as this will cause logging to be always enabled. Please remove, and update resource bundle description accordingly.